internal/api/handlers_test.go

```diff
@@ -349,7 +413,7 @@ func TestGoCertCertificatesHandlers(t *testing.T) {
 		},
 	}
 	for _, tC := range testCases {
-		t.Run(tC.desc, func(t *testing.T) {
+		t.Run(fmt.Sprintf("step %d: %s", tC.step, tC.desc), func(t *testing.T) {
```

Comment:
I would remove the step field and simply take this from the range:

```suggestion
    for step, tC := range testCases {
		t.Run(fmt.Sprintf("step %d: %s", tC.step, tC.desc), func(t *testing.T) {
```

Reply:
I liked having the steps because it helped scrolling to find the exact case that went wrong.

Reply:
Can't the same be done achieved with the description? This is making the file 26 lines longer at the moment, and will be a pain to maintain cleanly if we decide to remove a test case.

Reply:
that's not a bad idea. changed.